pennylane/plugins/default_qubit.py

```diff
@@ -455,9 +455,8 @@ def ev(self, A, wires):
         r"""Expectation value of observable on specified wires.
 
          Args:
-          observable      (str): name of the observable
+          A            (matrix): the observable
```

Comment:
type should be `array[float]` to be consistent with usage elsewhere in library. We sometimes put the shape in the following comment when it makes sense